AirCasting/CreateSessionViews/ConnectingABView.swift

```diff
@@ -19,22 +20,25 @@ struct ConnectingABView<VM: AirbeamConnectionViewModel>: View {
     
     
     var body: some View {
-        VStack(spacing: 50) {
-            ProgressView(value: 0.5)
+        VStack() {
+            ProgressView(value: 0.25)
```

Comment:
This value is actually decreasing progress bar. 🤔 Wasn't it good with 0.5?

Reply:
Yeah, it was. Thank you! my bad :/